### DISCUSSION TITLE
Mount not running when a lazy loaded component reacts to an event

### DISCUSSION THREAD

**Discussion by ollosh** [livewire/livewire#8430] [7. Bugs (archived)]
### Livewire version

v3.4.10

### Laravel version

v10.48.9

### Which PHP version are you using?

PHP 8.3

### Steps To Reproduce

1. Embed a lazy loaded component in your template. 
2. Add a method that listens to an event dispatched outside the component.
3. Create a mount method that defines some of the defaults
4. Dispatch an event without previously rendering the lazy loaded component
5. `mount` lifecycle hook in the component is not called, but just `render` which can produce some unexpected behaviour

### Repository or Wirebox URL

https://wirebox.app/b/ydm30

### Additional information

In the wirebox make sure to click on the FIRST CLICK ME button. If mount was running before the render when the lazy loaded component reacts to a dispatched event, the `clickedStatus` should be `clicked`, but instead it is empty. 

### Please confirm (incomplete submissions will not be addressed)

- [X] I have provided easy and step-by-step instructions to reproduce the bug.
- [X] I have provided code samples as text and NOT images.
- [X] I understand my bug report will be removed if I haven't met the criteria above.

**Comment by joshhanley** [comment]
@ollosh your wirebox is private, can you make it public? Thanks!

**Reply by ollosh** [reply]
@joshhanley Done!

**Comment by Catvert** [comment]
Hi,

I've got the exact same issue but with '#[Reactive]' variable.
If the component is lazy loaded (with a parent div x-show) and the reactive data is changed (on the parent) before the component is visible (and so loaded), then the mount function is never called.

Thanks for your help!

**Comment by eduPHP** [comment]
Hey there

I have a similar issue having two lazy components on different modals triggered in the same view, when one runs mount, the other doesn't

**Comment by Oreo-Monster** [comment]
Still an issue. 
My (ugly) workaround was in my event listener check if a property set in mount was set, and if not execute mount.
